test/cases/~hmr-resilience/webpack.config.js

```diff
@@ -0,0 +1,62 @@
+import { HotModuleReplacementPlugin } from 'webpack';
+
+import Self from '../../../src';
+
+module.exports = {
+  entry: './index.css',
+  mode: 'development',
+  devtool: false,
+  output: {
+    filename: '[name].js',
+  },
+  optimization: {
```

Comment:
Why it is necessary for test?

Reply:
output.filename is a leftover. I'll remove it.

Reply:
Removed the filename leftover, and left a comment explaining why the optimization chunk settings are being used.